### PR TITLE
fix archetype groupId and artifactId

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: samples/fibonacci-action/README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ This project is based on the Akka Serverless Maven archetype.
 
 ```shell
 mvn archetype:generate \
-  -DarchetypeGroupId=com.lightbend \
-  -DarchetypeArtifactId=maven-archetype-akkasls \
+  -DarchetypeGroupId=com.akkaserverless \
+  -DarchetypeArtifactId=akkaserverless-maven-archetype \
```
Comment (octonato) [inline samples/fibonacci-action/README.md:8]: This and the one below is not relevant. It's from the README in the samples. I'm just changing so I can rest at night. :-)

**Review by pvlugter — APPROVED** [review]

**Comment by pvlugter** [comment]
I'll publish just this change to the docs (so without other doc changes that also have code changes).

**Comment by pvlugter** [comment]
I've published this manually, along with changes from #367 and #376. Updated in both the dev and prod docs.

https://developer.lightbend.com/docs/akka-serverless/java/developer-tools.html

**Comment by patriknw** [comment]
@pvlugter Could you describe in RELEASING.md how to publish such doc hot fixes?

**Comment by pvlugter** [comment]
> @pvlugter Could you describe in RELEASING.md how to publish such doc hot fixes?

Sure, #378

**Pull request merged**